docs/Options.md

```diff
@@ -370,6 +371,19 @@ Docker section is shown only in directories that contain `Dockerfile` or `docker
 | `SPACESHIP_DOCKER_SYMBOL` | `🐳·` | Character to be shown before Docker version |
 | `SPACESHIP_DOCKER_COLOR` | `cyan` | Color of Docker section |
 
+### Ansible (`ansible`)
+
+Ansible section is shown only in directories that contain `*.yml` or `[.]ansible.cfg` files.
```

Comment:
Mentioning `*.yml` gives out impression this section would be displayed with all YAML files, Instead maybe mention as valid ansible playbooks ??

Reply:
or `.yaml`?

Reply:
reworded description, mentioned valid ansible playbooks